src/js/components/DropButton/__tests__/__snapshots__/DropButton-test.js.snap

```diff
@@ -3,15 +3,21 @@
 exports[`DropButton close by clicking outside 1`] = `
 <button
   aria-label="Open Drop"
-  class="StyledButton-gXzblK gTFGea"
+  class="StyledButton-gXzblK eVJqlx"
   label="Dropper"
   type="button"
 >
-  <span
-    class="StyledButton__StyledLabel-XZHBZ kthNMI"
+  <div
```

Comment:
I wonder if we should make the embedded Box use the 'span' tag instead of 'div'?